[Workers] wrangler install revision (#10369)
cloudflare/cloudflare-docs: content/workers/wrangler/install-and-update.md

$ wrangler version
```

## Update Wrangler

> To update Wrangler only in your current directory containing a `package.json`, run:

Author: ```suggestion
To update the version of Wrangler used in your project, run:
```
